lib/rules/disallow-multiple-var-decl.js

```diff
@@ -58,15 +69,25 @@ module.exports = function() {};
 module.exports.prototype = {
 
     configure: function(options) {
-        assert(
-            options === true ||
-            options === 'strict' ||
-            options === 'exceptUndefined',
-            this.getOptionName() + ' option requires a true value, "strict", or "exceptUndefined"'
-        );
-
-        this.strictMode = options === 'strict';
-        this.exceptUndefined = options === 'exceptUndefined';
+        if (typeof options === 'object') {
+            if (Array.isArray(options.allExcept)) {
+                this.exceptUndefined = options.allExcept.indexOf('undefined') > -1;
+                this.exceptRequire = options.allExcept.indexOf('require') > -1;
+            }
+
+            this.strictMode = options.strict === true;
+        } else {
```

Comment:
Non-blocker:
Not sure about this `else`-block. We can add a return in `if`. Personally, I'd swap these two blocks because `options` as object is not an exception in common.